### ISSUE TITLE
Is there anything about slot

### ISSUE THREAD

**Issue by wssnail** [dagger8224/dagger.js#1]
Is there anything about the use of slots and the scope of slots

**Comment by dagger8224** [comment]
> Is there anything about the use of slots and the scope of slots

No. "slot" is used to insert custom content in some frameworks such as Vue/Svelte. In dagger.js, you need to put the dynamic content insides the fields of $scope, and render it with "$html" or "$text" directive of the child module. It should be a more general solution than slots.

**Comment by wssnail** [comment]
> > Is there anything about the use of slots and the scope of slots
> 
> No. "slot" is used to insert custom content in some frameworks such as Vue/Svelte. In dagger.js, you need to put the dynamic content insides the fields of $scope, and render it with "$html" or "$text" directive of the child module. It should be a more general solution than slots.

Encourage component encapsulation?

If the component is encapsulated and for is encapsulated in the sub component, how can the display of a field in the sub component be modified in the parent component

**Comment by dagger8224** [comment]
> > > Is there anything about the use of slots and the scope of slots
> > 
> > 
> > No. "slot" is used to insert custom content in some frameworks such as Vue/Svelte. In dagger.js, you need to put the dynamic content insides the fields of $scope, and render it with "$html" or "$text" directive of the child module. It should be a more general solution than slots.
> 
> Encourage component encapsulation?
> 
> If the component is encapsulated and for is encapsulated in the sub component, how can the display of a field in the sub component be modified in the parent component

Please refer to this demo https://codepen.io/dagger8224/pen/vYdZKgx?editors=1100 to check out what happens if you modified the "message" field in the parent  scope.

**Comment by dagger8224** [comment]
@wssnail Since the version 0.9.17 dagger.js supports slot feature already. You may reference to https://codepen.io/dagger8224/pen/bGKPNwN for the usage of slot, thanks for your suggestion.
